cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/batch/BasicMapReduceContext.java

```diff
@@ -276,8 +274,12 @@ public void addOutput(Output output) {
    *
    * @return the OutputFormatProviders for the MapReduce job
    */
-  Map<String, OutputFormatProvider> getOutputFormatProviders() {
-    return ImmutableMap.copyOf(outputFormatProviders);
+  Collection<ProvidedOutputFormat> getOutputFormatProviders() {
```

Comment:
yeah, I first has only one method until I realized i need one just for logging purposes.